src/sentry/static/sentry/less/group-detail.less

```
@@ -522,6 +522,10 @@
       background-image: url(../images/logos/context/ubuntu.png);
     }
 
+    &.amazon .context-item-icon {
```

Trick now is to get the "known Amazon device ids" renamed to "amazon" to match this:

```suggestion
    &.amazon .context-item-icon {
```